internal/infra/pull/sql_datasource.go

```diff
@@ -80,10 +78,8 @@ func (ds *SQLDataSource) RowReader(source pull.Table, filter pull.Filter) (pull.
 	sql := &strings.Builder{}
 	sql.Write([]byte("SELECT * FROM "))
 	sql.Write([]byte(ds.tableName(source)))
-	sql.Write([]byte(" "))
-	if len(filter.Values()) > 0 {
-		sql.Write([]byte("WHERE "))
-	}
+	sql.Write([]byte(" WHERE "))
```

Comment:
Ici c'est une correction de bug qui a été écrasée par le merge je pense.

```suggestion
	sql.Write([]byte(" "))
	if len(filter.Values()) > 0 {
		sql.Write([]byte("WHERE "))
	}
```